src/wp-admin/includes/ms.php

```diff
@@ -1172,3 +1172,24 @@ function get_site_screen_help_sidebar_content() {
 		'<p>' . __( '<a href="https://developer.wordpress.org/advanced-administration/multisite/admin/#network-admin-sites-screen">Documentation on Site Management</a>' ) . '</p>' .
 		'<p>' . __( '<a href="https://wordpress.org/support/forum/multisite/">Support forums</a>' ) . '</p>';
 }
+
+
+/**
+ * Makes sure the passed $role is part of editable_roles
+ *
+ * @since 6.7.0
+ *
+ * @param string $role - name of the role
+ * @return void
+ */
+
+function ensure_editable_role( $role ) {
```

Comment:
The main change is to rename the function to use the `wp_` prefix 

```suggestion
/**
 * Stop execution if the role can not be assigned by the current user.
 *
 * @since 6.7.0
 *
 * @param string $role Role the user is attempting to assign.
 */
function wp_ensure_editable_role( $role ) {
```